src/sentry/release_health/metrics.py

```diff
@@ -376,7 +377,7 @@ def _count_users(total: bool, referrer: str) -> Dict[Any, int]:
         rv = {}
 
         for project_id, release in project_releases:
-            release_tag_value = indexer.resolve(org_id, release)
+            release_tag_value = indexer.resolve(org_id, release, use_case_id=UseCaseKey.PERFORMANCE)
```

Comment:
This module is `src/sentry/release_health/metrics.py`, why did you set it to `UseCaseKey.RELEASE_HEALTH`